components/src/dynamo/planner/core/base.py

```
@@ -313,6 +315,42 @@ async def _init_worker_info(self) -> None:
             self.decode_worker_info.model_name or self.prefill_worker_info.model_name
         )
 
+    def _resolve_runtime_namespace(self) -> str:
+        if hasattr(self.connector, "get_worker_runtime_namespace"):
+            try:
+                return self.connector.get_worker_runtime_namespace(  # type: ignore[attr-defined]
+                    self.namespace
+                )
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)